### ISSUE TITLE
Normalization for segmentation

### ISSUE THREAD

**Issue by powerLEO101** [Nischaydnk/RSNA-2023-1st-place-solution#3]
Hello!

Thank you again for posting the solution and answering my previosu questions. 

As I am trying to recreate and learn from your solution, I am confused by the normalizatoin of the prediction of 3d segmentatoin model. 

When training, the normalization is done in the dice loss function through smp, but when predicting, no normalizatoin was done and the outputs were still in logits. 

I understand that, if after sigmoid, it is OK to turn array to binary through final_outputs>0.5. But when there is no normalization, should the line be, instead, final_output > 0? 

I wonder if this is a mistake or intentionally?

https://github.com/Nischaydnk/RSNA-2023-1st-place-solution/blob/77ed554f80c626195d0e18adcd8a739b2a96ef86/Datasets/make_info_data.py#L179

Best

**Comment by powerLEO101** [comment]
And may I ask what is the final loss for 3d segmentation? Without TOTAL SEGMENTATOR data, I am stuck around 0.3.
